src/condor_io/condor_auth_ssl.cpp

```diff
@@ -1039,8 +1055,8 @@ Condor_Auth_SSL::server_verify_scitoken()
 			ss << auth << ",";
 		}
 		ad.InsertAttr(ATTR_SEC_LIMIT_AUTHORIZATION, ss.str());
-		mySock_->setPolicyAd(ad);
 	}
+	mySock_->setPolicyAd(ad);
```

Comment:
This changes when this code sets a policy ad at all.  Does the rest of the code handle the case where we have a policy ad but it doesn't contain `ATTR_SEC_LIMIT_AUTHORIZATION`?

(Is it strange that `ATTR_SEC_LIMIT_AUTHORIZATION` isn't in `SecMan::getSessionPolicy()`?)

Reply:
Did a quick check of the users of the policy ad -- everyone seems to handle a missing `ATTR_SEC_LIMIT_AUTHORIZATION` reasonably.

I think we've only added attributes one-by-one to `SecMan::getSessionPolicy()` as there are valid users.  Until we find some use case for needing the authorization limit outside the command handler, I guess it remains strangely missing?